app/webpack/stylesheets/_tables.scss

```diff
@@ -84,6 +84,30 @@ table {
         }
       }
     }
+
+    &.scheme_12_warning {
```

Comment:
since this shares the same  style of `injection-warning`, you should be able to create one and reuse for cav and schem 12 fees warning

Reply:
and [consistent naming too](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r476385484)

Reply:
👍 have combined css into single injection-warning

👍 have amended naming to scheme_12_fees_warning